Add customization options section (#127)
bump-sh/docs: src/_help/customization-options/logo-meta-images.md

---

> title: Logo and Meta images

Reviewer: We should add the "color" part to this page. So, maybe "Logo, Color and Meta image"? (do not forget to update the page name to reflect this change)

---

- TOC
{:toc}

## Logo and favicon

You can upload your own logo and favicon on Bump.sh to customize your documentation with your brand or project colors.

From the "Customize UI" tab in the documentation settings, you can add your own logo and favicon.

![](/images/help/logo-favicon.png)

> To provide the best image quality, it is recommended to adhere to the following dimensions for your logo: 140px*36px.

Reviewer: Are we sure of these dimensions (cc @thimy)? Also, should we mention that our renderer will try to adapt the best it can if other dimensions are provided?

Author: I've checked with @Thimy for these dimensions but I'll wait for her confirmation.
I didn't know that our renderer could do this, adding that to the next commit.

## Meta images

> A "meta image" typically refers to an image that is used as a preview image when a web page is shared on social media platforms and can be displayed in search engine results. You may have already heard it referred to as a "social media image." Meta images help make the links posted on social media platforms more attractive, especially when shared or displayed in posts.

Reviewer: I think we should start by mentioning that a meta image is generated by Bump.sh, by default. And change the order of the 2 sections.

Customizing the meta image is not something a lot of our customers do.

### Add your own image

From the "Customize UI" tab in the documentation settings, you can add your own meta image.

It is advisable not to exceed a width of 1200px and to maintain the height between 600px and 700px for better integration on most platforms.

![](/images/help/meta-image.png)

### Use the automated image generation

In case you don't have a visual you could use, Bump will automatically generate a meta image that will display your documentation name, logo and API type.

![](/images/help/meta-image-example.png)

> The social media image will be modified each time you change the name or the logo.
{: .info}

> First generation of this new social media image may take a bit, but any modification or change will start the generation process.

> {: .warning}

Reviewer: Can we try to avoid having several info/warning boxes following each other?

## Hub customization

All of these customization settings can be applied to all the documentation within a hub.

> These options are available directly in the "Customize UI" tab in the hub settings.

Reviewer: At the moment, it's under the "Default setting" part of the hub, but I think we should rename it, WDYT? (cc @thimy)

---
title: Logo and Meta images
---

- TOC
{:toc}

## Logo and favicon

You can upload your own logo and favicon on Bump.sh to customize your documentation with your brand or project colors.

From the "Customize UI" tab in the documentation settings, you can add your own logo and favicon.

![](/images/help/logo-favicon.png)

To provide the best image quality, it is recommended to adhere to the dimensions below:

> - Logo:

Reviewer 2: 140px*36px, cropped from potential paddings

> - Favicon:

Reviewer 2: You might not want to present the favicon in this list because we don't recommend a favicon size, as it may vary depending on the browser.
Ideally, one should provide different sized-favicons to fit every type of screens, but we don't support that.

## Meta images

A "meta image" typically refers to an image that is used as a preview image when a web page is shared on social media platforms and can be displayed in search engine results. You may have already heard it referred to as a "social media image." Meta images help make the links posted on social media platforms more attractive, especially when shared or displayed in posts.

### Add your own image

From the "Customize UI" tab in the settings of a documentation, you can add your own meta image.

It is advisable not to exceed a width of 1200px and to maintain the height between 600px and 700px for better integration on most platforms.

![](/images/help/meta-image.png)

### Use the automated image generation

> In case you wouldn't have a visual you could use, Bump will automatically generate a meta image. It displays your documentation name, its logo and its type.

Reviewer 2: ```suggestion
In the case you don't have a visual you could use, Bump will automatically generate a meta image that will display your documentation name, logo and API type.
```

![](/images/help/meta-image-example.png)

> The social media image will be modified each time you change the name or the logo.
{: .info}

> First generation of this new social media image may take a bit, but any modification or change will start the generation process.
{: .warning}

## Hub customization

> All of these customization settings can be applied to all the documentations within a hub.

Reviewer 3: ```suggestion
All of these customization settings can be applied to all documents within a documentation hub.
```

Author: Not sure of the usage here of "documents"

Author: But fixed the typo
